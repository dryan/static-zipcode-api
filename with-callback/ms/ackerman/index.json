{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["39735"], "name": "Ackerman", "counties": [{"fips": "019", "name": "Choctaw"}]}