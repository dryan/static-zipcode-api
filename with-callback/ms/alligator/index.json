{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["38720"], "name": "Alligator", "counties": [{"fips": "027", "name": "Coahoma"}, {"fips": "011", "name": "Bolivar"}]}