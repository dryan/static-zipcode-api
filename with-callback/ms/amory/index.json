{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["38821"], "name": "Amory", "counties": [{"fips": "095", "name": "Monroe"}]}