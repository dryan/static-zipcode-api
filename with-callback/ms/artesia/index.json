{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["39736"], "name": "Artesia", "counties": [{"fips": "087", "name": "Lowndes"}]}