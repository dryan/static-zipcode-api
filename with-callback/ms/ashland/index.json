{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["38603"], "name": "Ashland", "counties": [{"fips": "009", "name": "Benton"}, {"fips": "139", "name": "Tippah"}]}