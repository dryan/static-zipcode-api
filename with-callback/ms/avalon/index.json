{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["38930"], "name": "Avalon", "counties": [{"fips": "083", "name": "Leflore"}, {"fips": "001", "name": "Adams"}, {"fips": "015", "name": "Carroll"}]}