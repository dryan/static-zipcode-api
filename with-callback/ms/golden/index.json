{"region": {"fips": "28", "name": "Mississippi", "abbr": "MS"}, "postal_codes": ["38847"], "name": "Golden", "counties": [{"fips": "057", "name": "Itawamba"}, {"fips": "141", "name": "Tishomingo"}]}